yb-voyager/src/srcdb/oracle.go

```diff
@@ -512,7 +512,7 @@ func (ora *Oracle) GetColumnsWithSupportedTypes(tableList []sqlname.NameTuple, u
 		var unsupportedColumnNames []string
 		for i := 0; i < len(columns); i++ {
 			isUdtWithDebezium := (dataTypesOwner[i] == sname) && useDebezium // datatype owner check is for UDT type detection as VARRAY are created using UDT
-			if isUdtWithDebezium || utils.ContainsAnySubstringFromSlice(oracleUnsupportedDataTypes, dataTypes[i]) {
+			if isUdtWithDebezium || utils.ContainsAnySubstringFromSlice(OracleUnsupportedDataTypes, dataTypes[i]) {
```

Comment:
Even this is something we need to think about. This will only come up when some one runs export-data, not when assess-migration is run.  

In general, the list of unsupported data types depends on a bunch of factors: 
- pg_dump/debezium (for PG)
- offline or live (for both PG and oracle)
- forward migration or fall-forward or fall-back. 

Maybe for now it's okay to not mention the specifics, but let us discuss so that we are all on the same page about it.